libs/dh/admin/feature-user-management/src/lib/user-management-tabs/users-tab/dh-users-tab-table.component.html

```diff
@@ -14,28 +14,36 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<watt-table
-  #table
-  [columns]="columns"
-  [dataSource]="dataSource"
-  sortBy="name"
-  sortDirection="asc"
-  [sortClear]="false"
-  [resolveHeader]="translateHeader"
+<ng-container
+  *transloco="let t; read: 'admin.userManagement.tabs.users.table.columns'"
 >
-  <ng-container *wattTableCell="columns['name']; let element">{{
-    element.name
-  }}</ng-container>
-
-  <ng-container *wattTableCell="columns['email']; let element">{{
-    element.email
-  }}</ng-container>
-
-  <ng-container *wattTableCell="columns['phone']; let element">{{
-    element.phoneNumber | dhEmDashFallback
-  }}</ng-container>
-
-  <ng-container *wattTableCell="columns['status']; let element">{{
-    undefined | dhEmDashFallback
-  }}</ng-container>
-</watt-table>
+  <mat-table [dataSource]="dataSource">
```

Comment:
This is unacceptable! 🙃

Reply:
The minute `watt-table` supports custom data source, I'll convert this table to use our component. Promised 😄